src/selection-handler.js

```diff
@@ -339,7 +339,13 @@ Object.assign(SelectionHandler.prototype, require('./function-bind'), require('.
   },
 
   onPaste: function(ev) {
-    // TODO
+    if (ev.clipboardData.types.includes(TYPE)) {
```

Comment:
This should have a fallback so that you can paste plain text still. Maybe it defaults to Text block?

Reply:
This only matches when `TYPE` is matched as if it doesn't exist then the original paste functionality runs instead.